lib/travis/listener/app.rb

```diff
@@ -80,15 +82,15 @@ def handle_event
         return unless handle_event?
         debug "Event payload for #{uuid}: #{payload.inspect}"
         log_event(event_details, uuid: uuid, delivery_guid: delivery_guid, type: event_type, repository: slug)
-        Travis::Sidekiq::BuildRequest.perform_async(data)
+        Travis::Gatekeeper.push(Travis.config.gator.queue, data)
       end
 
       def handle_event?
         settings.events.include?(event_type)
       end
 
       def log_event(event_details, event_basics)
-        info(event_details.merge(event_basics).map{|k,v| "#{k}=#{v}"}.join(" "))
+        info(event_basics.merge(event_details).map{|k,v| "#{k}=#{v}"}.join(" "))
```

Comment:
Nope, that makes lots of sense to me, I forgot that Hashes are ordered in Ruby... I was just a bit surprised and was curious what the difference was, thanks for explaining! :heart: